content/blog/2021-11-30-november-21-community-gems.md

```diff
@@ -0,0 +1,188 @@
+┃ neutral:**Experiment**              ┃ metric:**step** ┃     metric:**acc** ┃ metric:**val_acc** ┃    metric:**loss** ┃ metric:**val_loss** ┃ param:**lr**    ┃ param:**momentum** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━╇━━━━━━━━━━┩
+│ **workspace**               │    **3** │ **0.91389** │    **0.87** │ **0.20506** │  **0.66306** │ **0.001** │ **0.09**     │
+│ **data-change**             │    **-** │       **-** │       **-** │       **-** │        **-** │ **0.001** │ **0.09**     │
+│ │ ╓ 9405575 [exp-54e8a] │    3 │ 0.91389 │    0.87 │ 0.20506 │  0.66306 │ 0.001 │ 0.09     │
+│ │ ╟ 856d80f             │    2 │ 0.90215 │ 0.87333 │ 0.27204 │  0.61631 │ 0.001 │ 0.09     │
+│ │ ╟ 23dc98f             │    1 │ 0.87671 │    0.86 │ 0.35964 │  0.61713 │ 0.001 │ 0.09     │
+│ ├─╨ 99a3c34             │    0 │ 0.71429 │    0.82 │ 0.67674 │  0.62798 │ 0.001 │ 0.09     │
+│ │ ╓ 3b3a2a2 [exp-23593] │    3 │ 0.86885 │    0.46 │ 0.31573 │   3.7067 │ 0.001 │ 0.09     │
+│ │ ╟ 93d015d             │    2 │ 0.83197 │ 0.41333 │ 0.36851 │   3.4259 │ 0.001 │ 0.09     │
+│ │ ╟ d474c42             │    1 │ 0.79918 │ 0.43333 │ 0.46612 │    3.286 │ 0.001 │ 0.09     │
+│ ├─╨ 1582b4b             │    0 │ 0.52869 │    0.39 │ 0.94102 │   2.5967 │ 0.001 │ 0.09     │
+└─────────────────────────┴──────┴─────────┴─────────┴─────────┴──────────┴───────┴──────────┘
+```
+
+### [What's the recommended way to remove data that has been imported using `dvc import`?](https://discord.com/channels/485586884165107732/485596304961962003/898462029650735134)
+
+Great question @MadsO!
+
+This works the exact same as when you've added data with `dvc add`. So to remove
+data, you would run this command:
+
+```dvc
+$ dvc remove
+```
+
+### [When using a CML, are GitHub Actions, GitLab, and BitBucket the only options for CI?](https://discord.com/channels/485586884165107732/728693131557732403/909847110306914345)
+
+Currently, `cml runner` does not support CircleCI or droneCI self–hosted runners
+and you would have to deploy them manually.
+
+You can still use `cml send-comment`, `cml pr`, and the other CML tools with any
+CI platform.
+
+Thanks for this awesome question @tpietruszka!
+
+### [When I run the `dvc remove` command, does it only remove .dvc files?](https://discord.com/channels/485586884165107732/563406153334128681/905382438786715648)
+
+A really good question from @flowy!
+
+That is correct. Running `dvc remove` only removes DVC tracked files and
+directories. It will also remove the entry from `.gitignore` and handles the
+`dvc.yaml`.
+
+If you still see a directory after you've removed a file from either DVC or Git,
+that is because neither removes empty directories from your filesystem.
+
+For example, if you run something like `dvc remove folder_name/file.dvc`, only
+the `.dvc` file will be removed. If there is nothing else in the folder, you'll
+be left with an empty directory. You can remove it with a command like:
```

Comment:
This still seems a little off to me since the question isn't specific to empty directories. What if there still is something in the folder? Normally, I would expect `folder_name/file` to exist if `folder_name/file.dvc` exists. In that case, `folder_name/file` would still exist after `dvc remove` unless you use `dvc remove --outs`.

Reply:
I think I cleared this up in the newest commit. Let me know if it makes sense!